README.md

```diff
@@ -51,7 +51,22 @@ Therefore, this application will not run into any issues if it is restarted, res
 | ENVIRONMENT              | If set to `dev`, will try to create the Kubernetes client using your local kubeconfig. Any other values will use the in-cluster configuration | no | `""` |
 | EXECUTION_INTERVAL | Duration to sleep between each execution in seconds | no | `20` |
 | EXECUTION_TIMEOUT | Maximum execution duration before timing out in seconds | no | `900` |
-
+| METRICS_PORT             | Port to bind metrics server to | no | `8080` |
+| METRICS                  | Expose metrics in Promtheus format at `:${METRICS_PORT}/metrics` | no | `""` | 
+
+## Metrics
+
+All the metrics are prefixed with `rolling_update_handler`.
```

Comment:
Can you just prefix the metrics in the table with that instead? It'll make searching using GitHub easier